9.0.3 Release

su-sws-jira opened the pull request:
# DO NOT DELETE

su-sws-jira committed:
9.0.3

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
HSD8-1037 D7 Image gallery importer (#859)

* HSD8-1037 D7 Gallery importer

* Generalized D7 importer config page

* fixed error

* Added menu link derivative for config pages

* cc fixes

* Correctly map the image alt text

* cc fix

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

Commit:
feat (STN-736): Address Vertical Timeline Bugs (#860)

* feat (STN-736): address vertical timeline bugs

- add toggle arrow icons to the expanded vertical timelines
- remove the toggle arrow icons from timeline itmes that do not have descriptions

* feat: add compiled CSS

pookmish committed:
HSD8-1016 Move the paragraph behaviors tabs to a different location

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
HSD8-1016 Improved javascript logic for paragraphs that dont have behaviors

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

Commit:
feat: update timeline component checkbox label for the expand / collapse functionality (#862)

Commit:
feat(STN-728): Gradient Hero Slider - Component Config (#863)

kaseybon committed:
docs: update lando troubleshooting section (#864)

Co-authored-by: Jon Oliver <jon@heysparkbox.com>

pookmish committed:
HSD8-1035 Hide "Select All" on content list

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

kaseybon committed:
STN-714: Fix npm start task (#865)

* chore: run build task on npm start

STN-714

* chore: npm audit

STN-714

pookmish committed:
Added layout paragraphs module to test out

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
Changed shield key

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
changed shield password for stage

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

Commit:
chore: update Gradient Hero Slider config (#870)

Updates the following configuration:
- Removes component from component, hero and row field selection.
- Adds duplicate option to slides
- Makes the slide field collapsed by default

kaseybon committed:
chore: backstop.js revisions (#871)

- remove backstop.json
- create task to compile backstop.json
- move staging basic auth credentials to an env file

STN-714

Commit:
chore: disable gradient hero slider on older themes only (#867)

pookmish committed:
Sparkbox Sprint 39

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
Fixed spotlight field help text

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
Refactored patterns to work better with layout paragraphs.

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

sherakama commented on composer.json:
```diff
@@ -189,6 +190,7 @@
         "drupal/key": "^1.14",
         "drupal/layout_builder_component_attributes": "^1.1",
         "drupal/layout_builder_modal": "^1.1",
+        "drupal/layout_paragraphs": "^1.0@beta",
```
Giving this a go eh? I am interested to hear about how it went at a later date.

pookmish replied:
yeah we're gonna experiment with it some to see how it does :smile:

pookmish committed:
Remove field permissions from contributor and site managers

pookmish committed:
Adjusted paragraph form displays

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
adjusted test for new permission

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
HSD8-1053 HSD8-1054 Provision biology and mtl sites

pookmish committed:
patch paragraphs module for edge case when paragraph type doesnt exist

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
Updated dependencies and configs

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
HSD8-1058 Add parameters to youtube videos

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
restored permission for row styles

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

pookmish committed:
codeclimate fixup

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs
